discord/ext/bridge/core.py

```diff
@@ -192,31 +201,170 @@ def after_invoke(self, coro):
+
+    Attributes
+    ----------
+    slash_variant: :class:`.SlashCommandGroup`
+        The slash command version of this command group.
+    ext_variant: :class:`.ext.commands.Group`
+        The prefix-based version of this command group.
+    subcommands: List[:class:`.BridgeCommand`]
+        List of bridge commands in this group
+    mapped: Optional[:class:`.SlashCommand`]
+        If :func:`map_to` is used, the mapped slash command.
     """
+    def __init__(self, callback, *args, **kwargs):
+        self.ext_variant = BridgeExtGroup(callback, *args, **kwargs)
+        self.slash_variant = SlashCommandGroup(self.ext_variant.name, *args, **kwargs)
+        self.subcommands: List[BridgeCommand] = []
+
+        self.mapped = None
+        if map_to := getattr(callback, "__custom_map_to__", None):
+            kwargs.update(map_to)
+            self.mapped = self.slash_variant.command(**kwargs)(callback)
+
+    @discord.utils.copy_doc(BridgeCommand.add_to)
+    def add_to(self, bot: ExtBot) -> None:
+        if self.slash_variant.subcommands:
+            bot.add_application_command(self.slash_variant)
+
+        bot.add_command(self.ext_variant)
+
+    def command(self, *args, **kwargs):
+        """A decorator to register a function as a subcommand.
+
+        Parameters
+        ----------
+        kwargs: Optional[Dict[:class:`str`, Any]]
+            Keyword arguments that are directly passed to the respective command constructors. (:class:`.SlashCommand` and :class:`.ext.commands.Command`)
+        """
+        def wrap(callback):
+            slash = self.slash_variant.command(*args, **filter_params(kwargs, brief="description"))(callback)
+            ext = self.ext_variant.command(*args, **filter_params(kwargs, description="brief"))(callback)
+            command = BridgeCommand(callback, slash_variant=slash, ext_variant=ext)
+            self.subcommands.append(command)
+            return command
+
+        return wrap
+
 
+def bridge_command(**kwargs):
+    """A decorator that is used to wrap a function as a bridge command.
+
+    Parameters
+    ----------
+    kwargs: Optional[Dict[:class:`str`, Any]]
+        Keyword arguments that are directly passed to the respective command constructors. (:class:`.SlashCommand` and :class:`.ext.commands.Command`)
+    """
     def decorator(callback):
         return BridgeCommand(callback, **kwargs)
 
     return decorator
 
 
+def bridge_group(**kwargs):
+    """A decorator that is used to wrap a function as a bridge command group.
+
+    Parameters
+    ----------
+    kwargs: Optional[Dict[:class:`str`, Any]]
+        Keyword arguments that are directly passed to the respective command constructors. (:class:`.SlashCommandGroup` and :class:`.ext.commands.Group`)
+    """
+    def decorator(callback):
+        return BridgeCommandGroup(callback, **kwargs)
+
+    return decorator
+
+
+def map_to(name, description = None):
```

Comment:
I don't get the point of this function. Why is it in `ext.bridge` if it's only supposed to be used on Slash Commands?

Reply:
Here it's supposed to map the primary command used by `Group` to a subcommand for slash so that it isn't wasted. Regular slash command groups don't have callbacks so if you mean that it could be used for regular groups that could be done in another pr.